scripts/buildDocs.sh

```diff
@@ -27,9 +33,26 @@ fi
 
 npm run generate-cli-doc
 
+# Setup Git config for mike & stash local settings to prevent overwrites
+LOCAL_GH_NAME=$(git config --default "" --local --get user.name)
```

Comment:
We only want to do this in CI, not when running the script locally. Otherwise we might create commits on behalf of OpenUI5 Bot (which happened already for this PR).

Isn't there a way to only have those settings within the GitHub workflow?

Reply:
(This comment is not for this exact line, but the overall handling and (re-)setting of the git user.name/email)

Reply:
Agree with that, but for some reason, Mike does not want to work with `git config --global` settings and for development, we need to explicitly provide `git config --local` user.name and user.email.

So, my point was, if we have set them up, then nothing happens, just the checks. But If they are not provided, to use CI defaults "for a while" and reset `git config --local` user.name and user.email. Basically, all this code is needed if we don't have `git config --local`

What do you think? Should we handle this or enforce developers to set those settings on their own?

Reply:
What about using a custom docker entrypoint as bash script within our image, that ensures the git config, and then runs mike?

But still, I don't quite understand why we can't set the `git config --local` within the workflow before running the container. As long as the `.git` folder is available within the mounted volume, the config should be available in the container.